packages/app/src/app/overmind/effects/api/types.ts

```diff
@@ -48,3 +48,14 @@ export type MetaFeatures = {
   loginWithGoogle?: boolean;
   loginWithWorkos?: boolean;
 };
+
+export type VMTier = {
+  name: string;
+  shortid: VMType;
+  cpu: number;
+  memory: number;
+  storage: number;
+  creditBasis: number;
+};
+
+export type VMType = 'vm-1' | 'vm-2' | 'vm-3' | 'vm-4' | 'vm-5' | 'vm-6';
```

Comment:
This is duplicated on `packages/app/src/app/constants.ts`

Reply:
Right, good catch. One is for deserializing the rest endpoint result, the other is for hardcoded data. This one from overmind can actually be just a string, but I went the extra length to define its possible values like this. I wouldn't change it now, but later we will get more data from BE and I will refactor the usage